identity-server/hosts/main/Program.cs

```diff
@@ -38,12 +38,19 @@
         .ConfigureServices()
         .ConfigurePipeline();
 
-    var usage = app.Services.GetRequiredService<LicenseUsageSummary>();
+    if (app.Environment.IsDevelopment())
+    {
+        app.Lifetime.ApplicationStopping.Register(() =>
+        {
+            var usage = app.Services.GetRequiredService<LicenseUsageSummary>();
+            Console.Write(Summary(usage));
+            {
```

Comment:
I'm not sure if this is a merge mistake, but there seems to be an extra `{}` scope here.

Reply:
Thanks for catching - fixed!

Reply:
This is the same in the other PR I just approved. Not a "huge" deal but you may want to update their too.